app/src/main/java/com/practicum/playlistmaker/ui/player/PlayerActivity.kt

```diff
@@ -0,0 +1,130 @@
+package com.practicum.playlistmaker.ui.player
```

Comment:
🍏 Пакет ui относится к слою presentation
```suggestion
package com.practicum.playlistmaker.presentation.ui.player
```